CONTRIBUTING.md

```diff
@@ -1,18 +1,103 @@
+# Contributor Guidelines
+
+## Developer Setup
+
 The easiest way to work on Ultra is to clone this repo and navigate to the
-`/workspace` directory. This is simple setup of an Ultra project, and imports
-Ultra from a relative URL.
+`/workspace` directory. It contains a simple webapp that imports Ultra from a
+relative URL.
```

Comment:
Is the point of the `/workspace` directory to have an app at the ready to test changes to Ultra? If I have this correct, it might be a good thing to explicitly say.

Reply:
Yes, the app is a good way to test changes. You can also run `deno task test:puppeteer` which exercises the same app.